ocaml/testsuite/tests/language-extensions/pprintast_unconditional.reference

```diff
@@ -13,17 +13,22 @@ modality_cstrarg:
       | Foo: global_ string * global_ string -> u 
   end
 
-modality_val: sig val t : string -> local_ string @@ foo bar end
+modality_val: sig val t : string -> string @ local @@ foo bar end
```

Comment:
To do this I think we'd have to add logic like "if the mode is one of the legacy mode strings, print it as a legacy mode. else, print it as an `@ mode`. I'm not sure doing that is worth it, especially considering `pprintast.ml` isn't necessarily supposed to be stable or anything, it's just supposed to roundtrip. Before my changes to `pprintast.ml`, if you tried to print the mode `foo` it would just fail inside the `legacy_modes` function.

Reply:
In our internal code there are code generated by `pprintast`, which (I'm afraid) might be read by humans. Those humans are not ready for the new syntax yet.

(If you are absolutely sure it won't be read by human, then it's fine).

Reply:
Resolved offline. Pushed the change to fix this.